### PR TITLE
return indicator emoji in bivariate_statistic query

### PR DESCRIPTION
**Pull request by sleeping-h**

### CODE REVIEW

File: src/test/java/io/kontur/disasterninja/service/layers/providers/BivariateLayerProviderTest.java [hunk 1]
```diff
@@ -65,14 +65,14 @@ private void init() {
         );
         List<BivariateLayerLegendQuery.Indicator> indicators = List.of(
                 new BivariateLayerLegendQuery.Indicator("Indicator",
-                        "area_km2", "Area", List.of(List.of("neutral"), List.of("neutral")),
+                        "area_km2", "Area", null, List.of(List.of("neutral"), List.of("neutral")),
```
Comment (gabriel-fallen) [inline src/test/java/io/kontur/disasterninja/service/layers/providers/BivariateLayerProviderTest.java:68]: It's nice everything works with `null` without an NPE, but maybe we should use an empty string in place of a missing emoji? I don't know for sure... What rules do we follow in such cases?

**Review by gabriel-fallen — APPROVED** [review]
Looks straightforward with no place for an error. 😸

File: src/test/java/io/kontur/disasterninja/service/layers/providers/BivariateLayerProviderTest.java (hunk 1 above)
Comment (albaranau) [reply src/test/java/io/kontur/disasterninja/service/layers/providers/BivariateLayerProviderTest.java:68]: From FE standpoint, I don't see a problem with null. At least we clearly see this data is missing, and it's easy to process

Comment (gabriel-fallen) [reply src/test/java/io/kontur/disasterninja/service/layers/providers/BivariateLayerProviderTest.java:68]: I've looked at the DB schema it seems to use `NULL` as a missing value as well. 👌

**Pull request merged**